tests/test_kubo_cas.py

```
@@ -0,0 +1,235 @@
+# async def test_kubo_default_urls(
+#     global_client_session, data: IPLDKind
+# ):  # Inject the session fixture
+#     """
+#     Tests KuboCAS using its default URLs and when None is passed for URLs,
+#     leveraging a globally managed aiohttp.ClientSession.
+#     """
+#     # Test Case 1: KuboCAS instantiated without explicit URLs (should use its defaults)
+#     # We pass the managed global_client_session to it.
+#     # KuboCAS itself is responsible for having default URLs if none are provided.
+#     async with KuboCAS(session=global_client_session) as kubo_cas_default:
+#         # print(f"Testing with default URLs: RPC={kubo_cas_default.rpc_base_url}, Gateway={kubo_cas_default.gateway_base_url}")
```

Pylint (3.3.7)

[convention] Line too long (129/100)

(C0301)